Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -35,9 +35,9 @@ namespace AudioCommon
 		std::string backend = SConfig::GetInstance().sBackend;
 		if (backend == BACKEND_OPENAL && OpenALStream::isValid())
 			g_sound_stream = new OpenALStream();
-		else if (backend == BACKEND_NULLSOUND && NullSound::isValid())
+		else if (backend.c_str() == BACKEND_NULLSOUND && NullSound::isValid())
```

Comment:
I only realized this after I wrote my defense.  I for some reason was thinking of the other comment.  I will use the string compare method instead.